docs/platform/api/linode-cli/index.md

```diff
@@ -249,6 +323,34 @@ Many other actions are available. Use `linode-cli linodes --help` for a complete
 
         linode-cli nodebalancers node-delete $nodebalancer_id $config_id $node_id
 
+Other actions are available. Use `linode-cli linodes --help` for a complete list.
+
+### Object Storage
+
+1.  List the current Object Storage Clusters available to use:
+
+        linode-cli object-storage clusters-list
+
+1.  Create a new Object Storage Key for your account:
+
+        linode-cli object-storage keys-create --label "my-object-storage-key"
+
+1.  List Object Storage Keys for authenticating to the Object Storage S3 API:
+
+        linode-cli object-storage keys-list
+
+1.  Update an Object Storage Key label:
+
+        linode-cli object-storage keys-update --keyId $key_id --label "my-new-object-storage-key"
+
+1.  Revoke an Object Storage Key:
+
+        linode-cli object-storage keys-delete --keyId $key_id
```

Comment:
This might not accept the --keyId argument. Might just needs keys-delete and the key ID.

Reply:
correct - removed